ra/registration-authority_test.go

```diff
@@ -218,9 +217,7 @@ func initAuthorities(t *testing.T) (*DummyValidationAuthority, *sa.SQLStorageAut
 
 	AuthzInitial.RegistrationID = Registration.ID
 
-	AuthzUpdated = AuthzInitial
-
-	AuthzFinal = AuthzUpdated
+	AuthzFinal = AuthzInitial
```

Comment:
I'm guessing we need to clean this up in a later PR because it was too much change in this one?